spec/GraphQLOverHTTP.md

```diff
@@ -1,57 +1,82 @@
-GraphQL Over HTTP
------------------
+## GraphQL Over HTTP
 
-Note: **Stage 0: Preliminary**
-This spec is under active development, and not ready for implementations yet. For more information, please see the [Roadmap](https://github.com/APIs-guru/graphql-over-http/blob/master/ROADMAP.md) or [how to get involved](https://github.com/APIs-guru/graphql-over-http/blob/master/INTERESTED_DEVELOPERS.md).
-You can find our community in the [graphql-over-http channel](https://graphql.slack.com/archives/CRTKLUZRT) on the [GraphQL Foundation slack](https://slack.graphql.org).
+Note: **Stage 0: Preliminary** This spec is under active development, and not
+ready for implementations yet. For more information, please see the
+[Roadmap](https://github.com/graphql/graphql-over-http/blob/master/ROADMAP.md)
+or
+[how to get involved](https://github.com/graphql/graphql-over-http/blob/master/INTERESTED_DEVELOPERS.md).
+You can find our community in the #graphql-over-http channel on the
+[GraphQL Foundation Discord](https://discord.graphql.org).
 
 ---
 
 # GraphQL over HTTP
 
 **Introduction**
 
-HTTP is the most common choice as the client-server protocol when using GraphQL because of its ubiquity.
-However the [GraphQL specification](https://graphql.github.io/graphql-spec/) deliberately does not specify the transport layer.
-
-The closest thing to an official specification is the article [Serving over HTTP](https://graphql.org/learn/serving-over-http/).
-Leading implementations on both client and server have mostly upheld those best practices and thus established
-a de-facto standard that is commonly used throughout the ecosystem.
-
-This specification is intended to fill this gap by specifying how GraphQL should be served over HTTP.
-The main intention of this specification is to provide interoperability between different client libraries, tools
-and server implementations.
+This specification details how libraries that wish to publish and consume
+GraphQL services over HTTP should do so in order to maximize interoperability
+between client libraries, tools, and server implementations. This specification
```

Comment:
```suggestion
This specification details how GraphQL should be served and consumed over HTTP
in order to maximize interoperability between clients, servers and tools. This specification
```

The spec concerns not only libraries, but also frameworks, applications, etc.

Reply:
What would be a good catch-all term? `software`?

Reply:
`implementors`?

I also like the suggested text.

Reply:
If we phrase it different, we can avoid having to find a term.

Reply:
My apologies I completely missed that there was a suggestion there somehow :sweat_smile: